src/UnwindRustSgxMem.S

```diff
@@ -0,0 +1,57 @@
+// (c) Fortanix, Inc.
```

Comment:
I'm fairly certain I already commented on this, but I can't find it. I don't think this file should be necessary. memcpy/memmove/memset should be provided by compiler_builtins

Reply:
Fixed. Pushing the changes